README.md

```diff
@@ -1,79 +1,45 @@
-          Simply name as the same tag name that you're doing and save it as an mp3
-          (e.g. anchor.mp3 for anchor). -->
-        <audio src="audio/tagname.mp3" controls></audio>
-      </h2>
-      <div class="base-words">
-        <div class="base-word-line">
-          <!-- <strong> includes the base word in English -->
-          English Base Word: <strong></strong>
-        </div>
-        <div class="base-word-line">
-          <!-- <strong> includes the base word in Spanish -->
-          Spanish Word Equivalent: <strong></strong>
-        </div>
-      </div>
-    </header>
-    <div class="definition">
-      <div class="definition-box definition-spanish">
-        <h4>Definici&oacute;n</h4>
-        <p></p>
-      </div>
-      <div class="definition-box definition-english">
-        <h4>Definition</h4>
-        <p></p>
-      </div>
-    </div>
-    <div class="how-to-remember">
-      <div class="how-to-remember-box how-to-remember-spanish">
-        <!-- Spanish block heading; no editing needed-->
-        <h4>&iquest;C&oacute;mo recordar?</h4>
-        <!-- Mnemonic Aid in Spanish -->
-        <p></p>
-      </div>
-      <div class="how-to-remember-box how-to-remember-english">
-        <!-- English block heading; no editing needed-->
-        <h4>How to Remember</h4>
-        <!-- Mnemonic Aid in English -->
-        <p></p>
-      </div>
-    </div>
-    <!-- <code> contains the code example. Use <strong> to select the keyword/specific tag -->
-    <pre class="code-example">
-      <code>
-        <!-- Place your awesome code example here :D -->
-      </code>
-    </pre>
-  </article>
-```
+# LMHT - HTML: Lenguaje de Marca de Hipertexto - Hyper Text Markup Language
+
+Welcome to [LMHT].  This is an open source code repository to help Spanish speakers translate and understand HTML tags.  Originally started as a project by Team Open Brace in the [YWebCA] Summer of 2015 class, this project is now being developed further as an open source project by [Adorable] to expand it's possibilities.  With your help, we can expand this into an excellent learning tool for bilingual programmers.  
+
+[LMHT]: http://lmht.github.io/
+[YWebCA]: http://ywebca.org/
+[Adorable]:  http://adorable.io/
+
+## Introduction
+
+The [LMHT page] provides a list of current HTML tags.  Select the tag you are looking for or begin typing it in the search box.  Once selected, a separate window will pop up containing the english base of the tag, the spanish equivalent, a definition in both english and spanish as well as audio recordings of each of these items.  Furthermore, there are helpful hints on how to remember what the tag does as well as examples of the code being used.
+
+[LMHT page]:  http://lmht.github.io/
+
+
+## Known Issues
+
+* Attributes: Currently, in the beginning stages, the focus has been on creating the primary html tags.  Eventually, this will expand to cover attributes associated with each of these tags.
```

Comment:
Use a consistent style for this phrase: `HTML tags` vs `html tags`. You have both.